### DISCUSSION TITLE
你们为什么这么不稳定？摆明了没有连接上的，你们还是显示的连接上，根本无法正常上网。你们什么时候能解决好？

### DISCUSSION THREAD

**Discussion by Appear6** [getlantern/lantern-forum-cn#1030] [General]
不知道别的地方什么情况，至少我这已经两个月了

**Comment by Appear6** [comment]
并且节点一直乱跳，有的节点本身是稳定的，用几分钟就跳到别的不稳定的地方去了

**Comment by bobobun212** [comment]
他们解决个J8，软件都不更新了，人都跑了吧

**Reply by MaxMKMax** [reply]
@bobobun212 您好. 我们上两周才更新而已.

**Reply by bobobun212** [reply]
更你妹，你自己用试试，看能连上吗

**Comment by MaxMKMax** [comment]
@Appear6 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Comment by MaxMKMax** [comment]
@Appear6 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

**Reply by Appear6** [reply]
79508782  我两周前就回复了邮件编号，但似乎是当时连着蓝灯所以连之前的回复都没发送成功.......

**Comment by wj19840706** [comment]
这么多年了，还这个样

**Comment by MaxMKMax** [comment]
@wj19840706 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。
